frontend/src/store/modules/sqlEditor.ts

```diff
@@ -53,14 +44,7 @@ export const useSQLEditorStore = defineStore("sqlEditor", {
   }),
 
   getters: {
-    connectionTreeByInstanceId(state): Partial<ConnectionAtom> {
-      const idx = state.connectionTree.findIndex((item) => {
-        return item.id === state.connectionContext.instanceId;
-      });
-
-      return idx !== -1 ? state.connectionTree[idx] : {};
-    },
-
+    // TODO: remove this after a refactor to <TableSchema>
```

Comment:
```suggestion
```

Reply:
I left this comment intentionally to remind me later.